src/components/ChecButton.vue

```diff
@@ -323,14 +323,14 @@ export default {
         @apply bg-#{map.get($config, 'default')} #{map.get($config, 'text')};
 
         &:not(.button--disabled):hover,
-        &:not(:disabled):hover {
+        &:not(:disabled):hover:enabled {
```

Comment:
This seems kind of strange to me, does the `:not(:disabled)` part not work?

Reply:
Oddly no, not sure why it disregards the links as span tags but works for buttons

Reply:
Interesting, it's related to the different tag types I guess. Maybe the `:disabled` pseudo-selector doesn't make sense for anchor tags.

Reply:
Yeah anchors don't have that selector, good point. Maybe we should use classes instead?

Reply:
🤷 I was just about to merge. It does look a bit weird having both, but CSS doesn't seem to have any issue and it works.

Reply:
That's fine